backend/controllers/orgUsers.js

```diff
@@ -9,7 +9,7 @@ const getAllOrgUsers = async (req, res) => {
 const createOrgUser = async (req, res) => {
   try {
     const orgUser = await OrgUser.create(req.body);
-    res.status(200).json(orgUser);
+    res.status(201).json({message: "Thanks! your account has been successfully created"});
```

Comment:
We should also add the below code after line 12
`orgUserId: orgUser._id`
because i.e once a user has been created we will need the orgUser ID i.e. to store it in when they create a request / listing and it will help with testing

Reply:
yeah would definitely be useful, however the user id gets created in the database and is not available to us until a find by user id method is implemented, so it needs another call to find this i'm pretty sure

Reply:
Ooo interesting, my bad, I thought once a user signs up we send the data to the backend which then a document gets created with the details i.e. Organisation Name, Email, etc which auto generate the ._id and we choose to send the databack `{message: "Thanks! your account has been successfully created"}` and along with the ._id but yeah can definitely look at this in the next sprint :)